website/docs/components/flyout/partials/guidelines/guidelines.md

```diff
@@ -114,11 +114,13 @@ The Flyout header features several properties to better communicate the purpose
 
 **With title icon**
 
-<Hds::Flyout::Header @icon="info" @onDismiss={{this.noop}}>Title</Hds::Flyout::Header>
+<!-- TODO: Temporary, replace with static image of Flyout Header -->
```

Comment:
@majedelass as part of refactors to the `Modal` & `Flyout` child components we'll have to replace these examples with images.

We'll need images for:
* Modal Header (with variations)
* Modal Footer (with variations)
* Flyout Header (with variations)
* Flyout Description (The Header together with the Description)
* Flyout Footer (with variations)

Basically we will need images to replace examples of the above child components on these Website pages:
* https://helios.hashicorp.design/components/modal (Guidelines tab)
* https://helios.hashicorp.design/components/flyout (Guidelines tab)

Reply:
@KristinLBradley can you sync with @majedelass to explain them what needs to be done here? thanks

Reply:
Resolving this (done in https://github.com/hashicorp/design-system/pull/1945/commits/f100a1a657e0abc3400dad3801aaddc9589296c2)